src/components/TopBar/components/SearchField/SearchField.scss

```diff
@@ -25,6 +24,7 @@ $stacking-order: (
 }
 
 .SearchField-newDesignLanguage {
+  z-index: z-index(modal, $fixed-element-stacking-order);
```

Comment:
Does this maybe need modal + 1? i.e if a modal is open will it interfere with the modal back drop at all? I'm guessing no but want to make sure

Reply:
Hmm, interesting thought. I can't see how two modals could be open at the same time, unless there is a bug (or one is triggered asynchronously). Are there any other cases?

Reply:
I was thinking that maybe a modal could be opened before the search field was activated. and the active modal background might interfere with the search field in some way. I just checked in core on one modal and it's not possible so I think you're good 👍